hack/aws-acceptance-test-cleanup/main.go

```diff
@@ -759,6 +764,68 @@ func destroyBackoff(ctx context.Context, resourceKind string, resourceID string,
 	}, backoff.WithContext(expoBackoff, ctx))
 }
 
+func cleanupIAMRoles(ctx context.Context, iamClient *iam.IAM) error {
+	var rolesToDelete []*iam.Role
+	rolePrefix := "consul-k8s-"
+	err := iamClient.ListRolesPagesWithContext(ctx, &iam.ListRolesInput{},
+		func(page *iam.ListRolesOutput, lastPage bool) bool {
+			for _, role := range page.Roles {
+				roleName := aws.StringValue(role.RoleName)
+				if strings.HasPrefix(roleName, rolePrefix) {
+					rolesToDelete = append(rolesToDelete, role)
+				}
+			}
+			return !lastPage
```

Comment:
Would be much helpful if you could add a one liner comment to the meaning of the `lastPage`